todo_app/model/viewModel.py

```diff
@@ -0,0 +1,36 @@
+from todo_app.data.itemstatus import ItemStatus
+
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    @property
+    def has_todo_items(self):
+        res = next((item for item in self._items if item.status == ItemStatus.TODO), None)
+        if res == None:
+            return False
+        else:
+            return True
+    @property
+    def has_doing_items(self):
+        res = next((item for item in self._items if item.status == ItemStatus.DOING), None)
+        print(res)
+        if res == None:
+            return False
+        else:
+            return True
+    @property
+    def has_finished_items(self):
+        res = next((item for item in self._items if item.status == ItemStatus.FINISHED), None)
+        if res == None:
+            return False
+        else:
+            return True            
+    @property
+    def todo_items(self):
+        return (item for item in self._items if item.status == ItemStatus.TODO)
```

Comment:
Is there any particular reason to be returning a generator rather than a list? If we did the code elsewhere would likely be more natural, e.g. being able to check lengths directly in your tests: 
`assert len(view.todo_items) == 1 ` vs `assert  sum(1 for _ in view.todo_items) == 1 `

That could just be unintentional but you've used () rather than []